<?xml version="1.0" encoding="UTF-8"?>

<launch>
    <arg name="start_frame" default="haro_base_link" />
    <arg name="end_frame" default="person_standing" />
    
    <node name="haro_to_object_tf_listener_node" pkg="haro_description" type="tf_haro_to_object_listener.py" args="$(arg start_frame) $(arg end_frame)" output="screen"/>
</launch>